app/services/contact_details/utilities/address_attributes.rb

```diff
@@ -0,0 +1,27 @@
+module ContactDetails
+  module Utilities
+    class AddressAttributes
+      def initialize(attributes)
+        @attributes = attributes
+      end
+
+      def attributes
+        uk_locale? ? @attributes.merge(reset_non_uk_attributes) : @attributes.merge(reset_uk_attributes)
+      end
+
+    private
+
+      def uk_locale?
+        @attributes[:locale_code] == "uk"
```

Comment:
There are several instances of the string "uk". We could replace these strings with a constant say `UK_LOCALE_CODE`

Reply:
Well, the locale code is meant to be an enum. Could I suggest we do it as a separate ticket to move this to a constant?